exe/appmap

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
```

Comment:
I think we should call this program something more descriptive, perhaps `appmap-agent-setup`, just to be very clear what it's for, and to avoid users calling it by accident by just typing `appmap` for whatever reason.